src/github.com/matrix-org/dendrite/mediaapi/writers/utils.go

```diff
@@ -0,0 +1,103 @@
+// Copyright 2017 Vector Creations Ltd
```

Comment:
Nit: `utils.go` is far too vague. This is dealing with files right? How about `fileutils.go`?